### ISSUE TITLE
[Sybil Report]

### ISSUE THREAD

**Issue by Tiodameiota1** [LayerZero-Labs/sybil-report#1096]
# Reported Addresses

0xc2c1c86d720288e044003c270bd83e467df42c92
0x52f1b88942c7c8949b0196cc9a6b82c677416a54
0x91bb7c14fd3c358cf3d5d361944e7e2e0a8e74e0
0xd7732e6cdabbe314c538fdcd625779fe1bcfac4e
0x8dc8a3801072aca2946abd6f39c5850692477898
0x279ef375805c2d80b276d87b42e44612da0a225c
0x2d4c91a748784c1bb30824417c11a208b8f8fead
0x122cce33242e6b18946e89d77969a6091a909dd0
0xb39c9e7c6a336f6c1f9f7f3f63fb91c805ec0658
0x9a5c974ff77c79c38c7682d0ae424dbb0eb049bb
0xca9ba83c58e883cbf08497b6af49710151490840
0x0b0f907a7e34a0ea286f95c72838d32563d098f5
0x31c057c29bc22208fc583810e63611af4c0fef6c
0x28afe8233a548b72d9359c59e954ae5a91ae9abf
0x7836a683c0c2dbcbebecffa9473818a927b3f7ff
0x745b2f1bb25accbb8e58aca96b6d937b2fd5deac
0x23d58e520e2c5f09b10910c2ca201113895516ca
0x6de25ff6cbb40687f313f02e75f91d49a61a3a1b
0x73db5bf2db042535ccc588865d7bfb0b38004349
0x4e8466dc8ac1430bd0e2e5739b83d5a9a1ca8377
0xb7f526fd3f8d91bc7fdc68e2a33704cb3bafcf72
0x3dc9777381a17971c36ae16c737e4e2d1558603f
0xcf7b65da0cc2b470e99f3b2619ad6fad6ddeadfb
0xa02fa79bd4e57c1eeb2872ebf4e46c438728d0db
0x03b6672c61d4da95df8c9cd54744ff0f8876ddc3
0x7e2c2fc99adf5a53ab17d01e88fc29505fdbf5d2
0x67e2d274ed62285e029d2123fc94fdd0a05af850
0x31fd43d8e04f4318bb615471bcb665e91aeb323b
0x92f9adb3c9a6edd4575e456192cade9922e513cf
0x0bb465a7e000fd9fd588189f916fbf3af9128b28
0x01408b9a0ca2fae0cb63a8253b348ad33b387e5c
0x0732d15db73b8c38a97aaa9e6547cdf0b4e568e7
0x89148e2ad5d075ef1ef614a39739f84e58c28bd6
0x868df19693a1eb0f9c252578f57b8219fbd01695
0x6b48c7c2d9eb1965030cc9eb28454e1faedf4493
0x67ede8969bd81a3e89d9138499b6d3948c7f62ff
0xd1178ca279cf5802067314ec9c0d0f07584dfed3
0x88587a9bbf3b095d2d3cc4b84c585876caeb3dda
0xaf45e37a437920bceb517a33dbea9d71b0a70676
0x9b8a6962bb050dd996e893f157ae59b351d309d7
0x4b6e16a6c66a3d66a68aa15dd938877cc4083369
0x77f50afa857f679e25d32412a92a4012990a5521
0xefd277373040938678dd7699f022acf5c430173b
0x6c72870afd82b0e712c261a14117c27625984f40
0xd67268af67e37365a98e818165fab3f08c1f1f7a
0xb2fd3f14ed4b8e81f434c03f4f4eda674e801828
0xff9c12782ff3fb91626dc136b5d248a3b0312ee1
0x079f9e8df56f45334e50e2d499781f482499ee3f
0x234c1bee05f77d29b0ed2a6a6ad92421e2dcca42
0x9b7a846fc4127c84a5bf6705d2f8f978b2baafc4
0x8782353bcb8f22044bca00399237e75beaf7d81b
0x2d94f461504a0e58fea186a3a5c75e2023d655aa
0x68bbf8b493acaaaf7be72257feb894236bc91004
0x599bfc80e5f78a467326e10e93213acbd4ca471c
0xd9e535f054bafb3b432be35ee5e1077fef427aa2
0x9e76d273f02b4f74785dec88dae5aaa5164e57fd
0x2fd23148aebb1ad2431f37797099405c65b79143
0x5f3fde609349d2f96ed1e8082e0820a821644111
0xccd1da78eb56752d0f19a8a4b9a6a5cc980dbb84
0xd4b3d388002f56747142d23fa0d1f7ed14f1fbe4
0x5d31edc2ea0fbd2a5281dc30e08b71830c7bf49a
0xb73123a147e994f70a974315070b6198e6e4d2b6
0x61c6b121bb3ecf1ab5a9589c2fa87e1c493883e1
0xad468093ad5258c877bc84e083696fc6b7aeb7b5
0x652e76d318c1e6a1d0b7b509b1a2282d9bd15a7d
0x130252b16d5b6170de0fc93234fdcf19138f7a22
0x164b3b7c9c684cae338d70c503e6a829767332ed
0x06b3ac9250bdd675370bcb07638036d35b843c6e
0xb1ddbe3fec08590d3866251b5d41cfbb9706f7c4
0x2646232b310296da0f5ca8110863cf7c47ad4928
0x96975c9e994852cf1df8543c5fe5292c84cee6cd
0x9cb087193f502660524d4432577f262d566de404
0x4df658f546e7296bad0a5cc0344c9ffa53770485
0x065e53cde466809e6a39fec474790a9b9c85c80b
0x18153e8b241ff681f9768f95c939bf4241a587e6
0x22f21beb769768a4c2f9e232fc9111e2ea8c1093
0x0e50f158385346bebcc86abfecab4306bb8b6e3e
0x93b5b48c5c6f06b5797a2a332e63727dcb18d93e
0x27706d66b67854bf52497743d907cdc7bddecdae
0x7bc4c492e2b1a07c13ebb8ce9c8f06fb1b5363a8
0x136715d8a74d9d7de36271737ea7315f0bb98f65
0xb926b021cd83736c119a483a3b806bfbcec74e01
0xf3ad641237f9d1fb0078dc1f3405c7a59a41fcbf

# Description

The networks used are the same, similar volume and similar transactions (all 14 to 20 transactions), almost all identical contracts... The first transaction of all wallets was from the Arbitrum network to Optimism using stargate, and all have 1 transaction of approximately 50k$ on arbitrum or optimism, then used the same protocols in the following months to stay active and look like a real user.

In short, they all have extremely similar transactions and are wallets entirely owned by a single person, as the final addresses for the money to be sent are the same.

Wallet 1 > Random Wallet > Wallet X (CEX probably)
Wallet 2 > Random Wallet > Wallet X (CEX probably)
Wallet 3 > Random Wallet > Wallet X (CEX probably)

# Detailed Methodology & Walkthrough

I used DUNE panels to check the similarity of days, networks, number of TX and volume. After that I did a manual check on the network scan and on the layerzero panel, where I found the similarity of all the Wallets.

# Reward Address (If Eligible)

0xbAf7e2a65F4694655447fB0D32838e7D142e4a34
